workflow/util/util_test.go

```diff
@@ -1064,7 +1064,7 @@ func TestRetryExitHandler(t *testing.T) {
 func TestFormulateRetryWorkflow(t *testing.T) {
 	ctx := context.Background()
 	wfClient := argofake.NewSimpleClientset().ArgoprojV1alpha1().Workflows("my-ns")
-	createdTime := metav1.Time{Time: time.Now().UTC()}
+	createdTime := metav1.Time{Time: time.Now().Add(-1 * time.Second).UTC()}
```

Comment:
I wondered actually why (but didn't verify since switching between windows/unix OS back and forth is a bit cumbersome ;)).
I might need to double check this.